src/proofPoints.ts

```diff
@@ -27,6 +27,42 @@ interface ProofPointIssueResult {
     proofPointObject: ProofPoint;
 };
 
+enum ProofPointValidity {
+    /**
+     * The proof point object is badly formed. The proof point is invalid.
```

Comment:
What might cause this? Can comment expand a bit more?

Reply:
well a proof point is a JSON document and according to the spec it needs certain fields to be present and some of the fields need to have certain values. Overall the document needs to be well formed. Not sure how to express that more succinctly

Reply:
ok no probs